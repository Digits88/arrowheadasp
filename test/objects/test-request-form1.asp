Testing simple form values.

Request.Form("valuea") = <%=Request.Form("valuea")%><br>
Request.Form("valueb") = <%=Request.Form("valueb")%><br>
