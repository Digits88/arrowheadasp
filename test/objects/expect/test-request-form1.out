Testing simple form values.

Request.Form("valuea") = one<br>
Request.Form("valueb") = two<br>
